b2sdk/b2http.py

```diff
@@ -241,10 +255,24 @@ def post_json_return_json(self, url, headers, params, try_count=5):
         :return: the decoded JSON document
         :rtype: dict
         """
+
+        # This is not just b2_copy_file or b2_copy_part, but it would not
+        # be good to find out by analyzing the url.
+        # In the future a more generic system between raw_api and b2http
+        # to indicate the timeouts should be designed.
+        timeout = self.TIMEOUT_FOR_COPY
+
         data = io.BytesIO(json.dumps(params).encode())
-        return self.post_content_return_json(url, headers, data, try_count, params)
+        return self.post_content_return_json(
+            url,
+            headers,
+            data,
+            try_count,
+            params,
+            force_timeout=timeout,
+        )
 
-    def get_content(self, url, headers, try_count=5):
+    def get_content(self, url, headers, try_count=20):
```

Comment:
I know it used to be like that, but I really don't like these literals just pasted here. Don't you think it would be better to use them as constants?

Reply:
if we use constants, people will start customizing it and we really don't want it to happen - the entire retry system is up for a rewrite

Reply:
if it's up for a rewrite then yeah, let's leave those literals as they are